deepchem/feat/graph_data.py

```diff
@@ -336,6 +336,7 @@ def __init__(self, graph_list: Sequence[GraphData]):
         ])
 
         # graph_index indicates which nodes belong to which graph
+        # TODO should we rename this as node_index?
```

Comment:
Can you remove TODO before merge?

Reply:
I have removed TODO but a question I have is should we rename `graph_index` as `node_index` because I think `node_index` is more descriptive. The variable here stores a mapping for every node and the graph to which the node belongs to.